analysers/analyser_merge_charging_station_FR.py

```diff
@@ -20,33 +20,147 @@
 ##                                                                       ##
 ###########################################################################
 
+import unittest
 from modules.OsmoseTranslation import T_
 from .Analyser_Merge import Analyser_Merge_Point, Source, CSV, Load_XY, Conflate, Select, Mapping
+import re
 
 
 class Analyser_Merge_Charging_station_FR(Analyser_Merge_Point):
+    def remove_trailing_zeros(input_string):
+        """
+        Removes dot and zeros at the end of a floating number typed in string
+        """
+        return str(input_string).replace('.00', '').replace('.0', '')
+
+    def map_wikidata_from_operator(self, str):
```

Comment:
Unused function?

Reply:
this is now an enreiched function with a small dictionnary